alloc/README.md

```diff
@@ -0,0 +1,27 @@
+# Alloc
+The allocator example demonstrates how to utilize the allocator feature when writing a contract. The `soroban-sdk` crate provides a lightweight bump-pointer allocator which can be used to emulate heap memory allocation in a Wasm smart contract.
+
+## Test
+For a quick test of the smart contract, run a test using the provided test file, `account/src/test.rs`. The test will just return a pass/fail result, but it’s a convenient way to check if the code works, without deploying and invoking the contract manually. The test file also demonstates how to invoke the smart contract. 
+
+From the root of the contract run this command:
+
+```
+cargo test
+```
+
+You should see the output:
+
+```
+running 1 test
+test test::test ... ok
+```
+
+See the main [README](../README.md) file for information about how to build and invoke the code using the CLI.
+
+## Relevant Links
+- [Open example in GitPod](https://gitpod.io/#https://github.com/stellar/soroban-examples/tree/v21.6.0)
```

Comment:
@carstenjacobsen shouldn't this be pointing at v22?

Reply:
I also think it's better to point to `main` branch instead of tag (so we don't need to update it in the future)

Reply:
@jeesunikim You are right, I didn't see that.
Gitpod as a service is going to be deprecated by April and we are going working to find another solution for DevRel. Maybe we should just remove the Gitpod link now for the examples?

Reply:
+1 on removing gitpod links (I personally never use it, just clone repo and run locally)